packages/pulpcore/python-ruamel-yaml/python-ruamel-yaml.spec

```diff
@@ -3,12 +3,12 @@
 %global srcname ruamel-yaml
 
 Name:           python-%{srcname}
-Version:        0.16.5
-Release:        2%{?dist}
+Version:        0.16.10
```

Comment:
can you please revert the ruamel changes, I'd prefer to look at them independently as this package is a bit special :(